src/simulator/diff_drive_model.cpp

```diff
@@ -54,19 +54,21 @@ CONFIG_FLOAT(angular_odom_scale, "angular_odom_scale");
 CONFIG_STRING(drive_topic, "drive_callback_topic");
 CONFIG_STRING(odom_topic, "diff_drive_odom_topic");
 
-DiffDriveModel::DiffDriveModel(const vector<string>& config_files, ros::NodeHandle* n) :
+DiffDriveModel::DiffDriveModel(const vector<string>& config_files, 
```

Comment:
As of this PR only the diff drive model will properly handle the topic prefix and multi robot functionality.